doc/source/workflow.rst

```diff
@@ -207,6 +214,474 @@ KIWI:
 
    #. **Container Image**: Base images for the Docker engine.
 
+Components of an Image Description
+----------------------------------
+
+A KIWI image description can composed by several parts. The main part is
+the KIWI description file itself, the :file:`config.xml` file. The
```

Comment:
@schaefi yes, this is explained in a note in `Building Image` section. But you are right it makes a lot of sense to remind it here. I'll place a link or add a comment here.